website/docs/components/form/checkbox/partials/code/how-to-use.md

```diff
@@ -238,7 +238,7 @@ Use the standard HTML `checked` attribute to mark the input as checked.
 
 #### Indeterminate
 
-In addition to the checked and unchecked states, a checkbox can be in an indeterminate state, also referred to as partially checked state. A common use case for this state is when a parent checkbox allows the user select multiple children checkboxes at once.
+In addition to the checked and unchecked states, a checkbox can be in an indeterminate state, also referred to as partially checked state. A common use case for this state is when a parent checkbox allows the user select multiple children checkboxes at once. The visual appearance of the checkbox is determined by the component based on the `indeterminate` attribute using JavaScript.
```

Comment:
I thought it was an Ember thing, but actually it's not. I suggest we link to the MDN documentation, to point directly to the documentation for this special attribute.
```suggestion
In addition to the checked and unchecked states, a checkbox can be in an indeterminate state, also referred to as partially checked state. A common use case for this state is when a parent checkbox allows the user select multiple children checkboxes at once. The visual appearance of the checkbox is determined by the component based on the [`indeterminate` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox#indeterminate_state_checkboxes) using JavaScript.
```

Reply:
well, the JavaScript bit is actually part of Ember's way of getting and setting such attributes following the standard – we didn't have to write any line of code for this to happen